pkg/gui/keybindings.go

```diff
@@ -125,6 +125,12 @@ func (gui *Gui) GetKeybindings() []*Binding {
 			Modifier:    gocui.ModNone,
 			Handler:     gui.handleCommitPress,
 			Description: gui.Tr.SLocalize("CommitChanges"),
+		}, {
+			ViewName:    "files",
+			Key:         'M',
```

Comment:
I see that the 'A' keybinding is used to abort merges, but this seems like an operation that wouldn't be used often. What are your thoughts on using shift+M for `abort merge` and shift+A for `amend commit`?

Reply:
Yeah that makes sense. I'll switch it.

Reply:
Done.